### ISSUE TITLE
Windows Service starts then stops with no output error

### ISSUE THREAD

**Issue by StormProductionsMusic** [jmdaweb/NVDARemoteServer#40]
Hi,

Windows Service seems to start then stop with no error from the console. 
On Windows Services screen, when I click Start, I get the following message:
Services  dialog  The NVDARemote relay server service on Local Computer started and then stopped. Some services stop automatically if they are not in use by other services or programs.

It worked fine until I changed something in the config file - I did check now and everything seems okay, I haven't messed up anything there.

Any way I can get this fixed?

Thanks.

**Comment by jmdaweb** [comment]
Hi,
The Windows service never displays errors in a directly visible way. Take a look at your NVDARemoteServer.log file. Also, try running the server in debug mode.
Regards.

**Comment by StormProductionsMusic** [comment]
I discovered the problem.
Aparently, it doesn't like if you change the logfile path, for some weird reason. I tried all type of combinations possible, but no luck. If I place an # before logfile, everything worls, but if I remove it (from the Windows part of course), it freaks up everything and the service won't start.
Anyway, closing this for now, but feel free to reply in case you know what happens.

Thanks for your help.
